bookie/tests/test_auth/test_api_base.py

```diff
@@ -123,3 +123,28 @@ def test_login_null(self):
         self.assertTrue(
             'Failed login' in str(res),
             "Should have 'Failed login' in the resp: " + str(res))
+
+    def test_redirect_for_authenticated_user(self):
```

Comment:
given the updated sanity check below those should have matching tests to make sure we verify we force a login to invalid users.